plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/callables/CallableSignature.kt

```diff
@@ -15,6 +15,8 @@ interface CallableSignature {
     val receiverType: TypeEmbedding?
     val paramTypes: List<TypeEmbedding>
     val returnType: TypeEmbedding
+    val returnsUnique: Boolean
```

Comment:
One thing to think about: we have type information available for everything, we could make uniqueness a type modifier.  WDYT?

Reply:
It can work for function signatures. However I suspect that for the rest (variable and paths) uniqueness should be a function taking a context as argument. Probably what to do will be clearer when we'll introduce contexts.